src/libslic3r/GCode.cpp

```diff
@@ -6667,4 +6804,4 @@ void GCode::ObjectByExtruder::Island::Region::append(const Type type, const Extr
 // a single object, or for possibly multiple objects with multiple instances.
 
 
-} // namespace Slic3r
```

Comment:
It's just a comment, but I am curious what caused this diff?

Reply:
I had seen the last comment formatted similarly in Bambu Studio's GCodeProcessor.hpp file, so I just carried it over to the files while I was there.